### ISSUE TITLE
Does not read the IP from `x-forwarded-for`

### ISSUE THREAD

**Issue by bukowskiadam** [gaurishhs/elysia-ip#30]
Hey, I've spent an hour debugging my app. It turns out there is a regression introduced in this commit: https://github.com/gaurishhs/elysia-ip/commit/fb5db36d90398db677637ed5eb1d4239a3a6c7fc

Now if you go with default config, the `x-forwarded-for` header has no effect, because of this line:
https://github.com/gaurishhs/elysia-ip/blob/ed24145e06b8ec9e4a105482c78725d2a4597c53/src/services/getip.ts#L15

which checks if `!checkHeaders` which is false, because `checkHeaders === headersToCheck` which is truthy.

File to reproduce:
```ts
import { Elysia } from 'elysia';
import { ip } from 'elysia-ip';

const app = new Elysia()
  .use(ip({ headersOnly: true }))
  .get('/', ({ ip }) => `This is the ip: "${ip}"`);

const request = new Request('http://localhost/');

// This should be the request IP
request.headers.set('x-forwarded-for', '1.1.1.1');

// but this is the returned IP
request.headers.set('x-real-ip', '2.2.2.2');

app
  .handle(request)
  .then((response) => response.text())
  .then(console.log);
```
Put it in the ts file and run `bun test.ts`

**Comment by gaurishhs** [comment]
It was an extra `!` I've fixed it. `x-forwarded-for` is to be checked only when using default config (i.e. no user provided headers) and this is the header which has to be checked first

**Comment by gaurishhs** [comment]
Fixed in v1.0.8 Please upgrade. Thanks!

**Comment by bukowskiadam** [comment]
Thanks! That was the other solution I was thinking about. But I thought that there is a special treatment handling comma separated list of IP, so I thought that it's better to always check it.
